x-pack/plugins/security_solution/server/endpoint/routes/resolver/docs/README.md

```diff
@@ -0,0 +1,214 @@
+# Resolver Backend
+
+This readme will describe the backend implementation for resolver.
+
+## Ancestry Array
+
+The ancestry array is an array of entity_ids. This array is included with each event sent by the elastic endpoint
+and defines the ancestors of a particular process. The array is formatted such that [0] of the array contains the direct
+parent of the process. [1] of the array contains the grandparent of the process. For example if Process A spawned process
+B which spawned process C. Process C's array would be [B,C].
```

Comment:
:red_circle: You mean `Process C's array would be [B,A].` I think.

Reply:
Yep, thanks for catching that